src/main/java/uk/gov/hmcts/ccd/domain/service/createevent/ClassifiedCreateEventOperation.java

```diff
@@ -29,7 +29,7 @@ public CaseDetails createCaseEvent(String caseReference,
         final CaseDetails caseDetails = createEventOperation.createCaseEvent(caseReference,
                                                                            content);
         return Optional.ofNullable(caseDetails)
-                       .flatMap(classificationService::applyClassification)
+                       .flatMap(details -> classificationService.applyClassification(details, true))
```

Comment:
We're passing `create` as `true` here, but the case already exists when creating an event - is this correct?

Reply:
remove this now